presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -173,6 +175,13 @@ else if (config.getPinGlueClientToCurrentRegion()) {
             asyncGlueClientBuilder.setCredentials(credentialsProvider);
         }
 
+        if (config.getAwsAccessKey().isPresent() && config.getAwsSecretKey().isPresent()) {
+            AWSCredentialsProvider credentialsProvider = new AWSStaticCredentialsProvider(
+                    new BasicAWSCredentials(config.getAwsAccessKey().get(), config.getAwsSecretKey().get())
+            );
+            asyncGlueClientBuilder.setCredentials(credentialsProvider);
+        }
```

Comment:
Ideally, the options: access keys and IAM role -- they should be mutually exclusive.
For now, let's at least be consistent with how S3 gets configured (access keys are checked first, before IAM role)
Please move this before `if (config.getIamRole().isPresent())` block.